manageiq-providers-ibm_cloud.gemspec

```diff
@@ -26,6 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "ibm_cloud_databases", "~> 0.1"
   spec.add_dependency "ibm_cloud_global_tagging", "~> 0.1"
   spec.add_dependency "ibm_vpc", "~> 0.1"
+  spec.add_dependency "rest-client", ">= 2.0.0"
```

Comment:
Best to stay away from `>=` for gem deps because that would include major version changes.

The core Gemfile has `"~>2.1.0"` if you want to open it up a bit in the plugin (not a bad idea) you could do `~> 2.1` here which would allow for minor version bumps